website/docs/visual-testing/integrate-with-app-percy.md

```diff
@@ -0,0 +1,111 @@
+---
+id: integrate-with-app-percy
+title: Run visual tests on your mobile apps with App Percy
+---
+
+## Introduction
+
+[Percy](https://percy.io/?utm_source=webdriverio&utm_medium=partnered&utm_campaign=documentation) is an all-in-one visual testing and review platform. It captures screenshots, compares them against the baseline, and highlights visual changes. With increased visual coverage, teams can deploy code changes with confidence with every commit.
+
+WebdriverIO supports cross-browser visual testing natively using App Percy. You can use App Percy for visual testing of native mobile applications.
+The benefits of utilizing App Percy for visual testing include the following:
+
+- Consistency: Promotes consistent user experience by identifying visual discrepancies early in the development process.
+- Efficiency: Improves efficiency by reducing the time and effort required to manually spot visual regressions.
+- Integrations: Percy integrates with popular tools and services like GitHub, GitLab, Bitbucket, and more.
+- Collaboration: Improves collaboration between developers, designers, and QA teams by providing a visual representation of changes.
+- Prevent regression: Prevents you from experiencing unintended visual regressions.
+
+## How does Percy work?
+
+Percy compares new snapshots against relevant baselines to detect visual changes. Percy manages baseline selection across branches so your tests are always relevant. If visual changes are detected, Percy highlights and groups the resulting diffs for you to review.
```

Comment:
This whole section is almost 1:1 the same to the other document. Let's maybe have a general page for Percy and then create 2 documents going into the specifics of using Percy for web environments and one for mobile applications

Reply:
Done. I have incorporated the suggested comments.